cli/app.go

```diff
@@ -62,7 +62,6 @@ var app = &cli.App{
 		&cli.StringFlag{
 			Name:   baseURLFlag,
 			Hidden: true,
-			Value:  "https://app.viam.com:443",
```

Comment:
Why was this removed? Did we just replace this with the `defaultBaseURL` variable?